lightningd/pay.c

```diff
@@ -1343,6 +1343,7 @@ static struct command_result *json_sendonion(struct command *cmd,
 			 /* FIXME: parameter should be invstring now */
 			 p_opt("bolt11", param_invstring, &invstring),
 			 p_opt_def("amount_msat", param_msat, &msat, AMOUNT_MSAT(0)),
+			 p_opt_def("total_amount_msat", param_msat, &total_msat, AMOUNT_MSAT(0)),
```

Comment:
You can't add new arguments in the middle, as parameters can be specified as an array.  New parameters need to go at the end.